### ISSUE TITLE
[Bug]: ALPHA: "The Bus" level unbeatable

### ISSUE THREAD

**Issue by McModknower** [Turing-Complete-Game/Suggestions-and-Issues#65]
### Discord Thread

https://discord.com/channels/828292123936948244/1264949426628333588

### What happened?

In the current alpha version 0.1073 Beta i get the message "Copy the value from input -1 to output 0" when trying to run my solution.

according to https://discord.com/channels/828292123936948244/899582781271736361/1264945480043991121 the level worked in 0.1068

The debug.log might not be relevant since it didn't change when reattempting that level.

### Version

0.1073 Beta

### What OSes are you seeing the problem on?

Linux

### Relevant `debug.log` output, search next to the exectuable

```shell
Godot Engine v3.4.2.stable.custom_build - https://godotengine.org
Godot Engine v3.4.2.stable.custom_build - https://godotengine.org
Godot Engine v3.4.2.stable.custom_build - https://godotengine.org
OpenGL ES 3.0 Renderer: NVIDIA GeForce GTX 1660 SUPER/PCIe/SSE2
OpenGL ES Batching: ON
 
Turing Complete 0.1073 Beta on X11
ERROR: Failed to invoke Nim procedure update_paste_selection: expected 0-0 arguments, but got 1
   at:  (godotmacros.nim:497) - Failed to invoke Nim procedure update_paste_selection: expected 0-0 arguments, but got 1
ERROR: Failed to invoke Nim procedure show_paste: expected 1-1 arguments, but got 0
   at:  (godotmacros.nim:497) - Failed to invoke Nim procedure show_paste: expected 1-1 arguments, but got 0
```

**Comment by McModknower** [comment]
~~This seems to be fixed in 0.1076 beta. Should i close this issue then?~~
Not fully fixed, see attached image
![image](https://github.com/user-attachments/assets/be40f18b-52d1-4291-92f1-cf7f08b31457)

**Comment by Stuffe** [comment]
Now it is fixed
